Automattic/wp-calypso
# Gutenboarding v9: enable Feature Picker in production and bump AB datestamp

#### Changes proposed in this Pull Request

* This releases the feature picker to production and bumps the AB datestamp. 

#### Testing instructions

Create a site using `calypso.live` below and you should see the feature picker after you select a font pairing.

Fixes https://github.com/Automattic/wp-calypso/issues/44428
